test/OpenTelemetry.Exporter.Geneva.Tests/UnixUserEventsDataTransportTests.cs

```diff
@@ -143,6 +144,106 @@ public void UserEvents_Disabled_Success_Linux()
         }
     }
 
+    [Fact(Skip = "This would fail on Ubuntu. Skipping for now. See issue: #2326.")]
+    public void UserEvents_Logs_Success_Linux()
+    {
+        EnsureUserEventsEnabled();
+
+        var listener = new PerfTracepointListener(
+            "MicrosoftOpenTelemetryLogs_L4K1",
```

Comment:
is "L4" hardcoded? or its based on severity?
Also, is K1 fixed with no ability to change?

Reply:
Both numbers can change.

Description:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/c2eb2e50262f7eefbc50e65fb3906105cca1890d/src/OpenTelemetry.Exporter.Geneva/External/LinuxTracepoints-Net/Provider/EventHeaderDynamicTracepoint.cs#L13-L15

Constructor:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/c2eb2e50262f7eefbc50e65fb3906105cca1890d/src/OpenTelemetry.Exporter.Geneva/External/LinuxTracepoints-Net/Provider/EventHeaderDynamicTracepoint.cs#L62

For now I've hard-coded all these values, but at least EventLevel will be dynamic as I make code changes later.